src/entity/Action.ts

```diff
@@ -0,0 +1,32 @@
+import { Session } from './Session';
+import { Command } from './Command';
+import { Exception } from './Exception';
+
+import {
+	Column,
+	Entity,
+	ManyToOne,
+	OneToMany,
+	PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity()
+export class Action {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@ManyToOne(() => Session, session => session.actions)
+	session: Session;
+
+	@OneToMany(() => Command, command => command.action)
```

Comment:
Why is this a one to many relationship? commands and exceptions should inherit action